sql/updates/world/3.3.5/9999_99_99_99_world_COT_STRATHOLME.sql

```diff
@@ -0,0 +1,319 @@
+-- Caverns of Time: Stratholme rework
+SET @DAY = 86400;
+SET @CGUID = 700300; -- creature GUIDs (4 creature);
+DELETE FROM `creature` WHERE `guid` BETWEEN @CGUID+00 AND @CGUID+03;
```

Comment:
just had a thought: it's better to delete by guid and id so we get an error if we are deleting by mistake another creature. this is not just for this PR but for everyone writing sql scripts